public/app/common/components/list/viewmodels/test/component-viewmodel.spec.coffee

```diff
@@ -0,0 +1,23 @@
+ComponentViewModel = require '../component.viewmodel'
```

Comment:
Sorry, I've added a couple of comments for this file to the commit itself rather than here. Please refer to 8722ad06